src/ucs/datastruct/bitmap.h

```diff
@@ -21,6 +21,11 @@ BEGIN_C_DECLS
 typedef uint64_t ucs_bitmap_word_t;
 
 
+typedef struct {
+    uint8_t bits[1];
+} ucs_bitmap8_t;
```

Comment:
i mean, can't we use just uint8_t in rdma_cm?

Reply:
I use ucs_bitmap_t to emphasize that it is a bitmap and not a number